marinerescue-frontend/src/pages/AboutPage.js

```diff
@@ -1,11 +1,121 @@
 import React from 'react';
 
+import { Link } from "react-router-dom";
+
+import './AboutPage.scss';
+
+
 class AboutPage extends React.Component {
     render() {
         return (
-            <div>
-                <h1>About</h1>
-            </div>
+            <main>
+                <div class="navbar">
+                    <Link to= "/">
+                    <div class="navbar-logo">
+                        <img src="images/logo.png"></img>
+                    </div>
+                    </Link>
+                    <Link to="/leaderboard">Leaderboard</Link>
+                    <Link to="/about">About</Link>
+                    <Link to="/parents">Parents</Link>
+                    <a href="#playnow"><button id="nav-bar-button">Play Now</button></a>
+                    <a href="profile">Profile</a>
+                    {/* <div class="navbar-logo">
+                        <img src="images/profile-logo.png"></img>
+                    </div> */}
+                </div>
+
+                <div class="section">
+                    <div class="row">
+                        <div class="column">
+                            <h1>- Our Goal - </h1>
+                            <h2>Marine Rescue is a platform for elementary students to learn about debris' impacts on marine wildlife.</h2>
+                            <br />
+                            <h2>Through interactive educational stories and activities, students contribute to citizen science projects so that they can develop a sense of connection to the natural world through active learning while meaningfully contributing to science research. From our user research when developing this project, we found a lack of designated science class time in schools and a lack of curriculum for parents teaching science at home. This inspired us to create Marine Rescue as an accessible and hands-on learning experience for students to learn marine science outside of the classroom. </h2>
```

Comment:
Using heading tags for non-headings is bad practice and poor for accessibility, could you make this into <p>s? If you just want a bigger font, use CSS :)
```suggestion
                            <p>Marine Rescue is a platform for elementary students to learn about debris' impacts on marine wildlife.</p>
                            <p>Through interactive educational stories and activities, students contribute to citizen science projects so that they can develop a sense of connection to the natural world through active learning while meaningfully contributing to science research. From our user research when developing this project, we found a lack of designated science class time in schools and a lack of curriculum for parents teaching science at home. This inspired us to create Marine Rescue as an accessible and hands-on learning experience for students to learn marine science outside of the classroom. </p>
```